plugins/net_plugin/net_plugin.cpp

```diff
@@ -4285,15 +4300,24 @@ namespace eosio {
          ( "p2p-disable-block-nack", bpo::value<bool>()->default_value(false),
             "Disable block notice and block nack. All blocks received will be broadcast to all peers unless already received.")
          ( "p2p-auto-bp-peer", bpo::value< vector<string> >()->composing(),
-           "The account and public p2p endpoint of a block producer node to automatically connect to when it is in producer schedule proximity\n."
-           "  Syntax: account,host:port\n"
-           "  Example,\n"
-           "    eosproducer1,p2p.eos.io:9876\n"
-           "    eosproducer2,p2p.trx.eos.io:9876:trx\n"
-           "    eosproducer3,p2p.blk.eos.io:9876:blk\n")
-         ("p2p-producer-peer", boost::program_options::value<vector<string>>()->composing()->multitoken(),
-           "Producer peer name of this node used to retrieve peer key from on-chain peerkeys table. Private key of peer key should be configured via signature-provider.")
-         ( "agent-name", bpo::value<string>()->default_value("EOS Test Agent"), "The name supplied to identify this node amongst the peers.")
+           "The account and public p2p endpoint of a block producer node to automatically connect to when it is in producer schedule. Not gossipped.\n"
+           "  Syntax: bp_account,host:port\n"
+           "  Example:\n"
+           "    producer1,p2p.prod.io:9876\n"
+           "    producer2,p2p.trx.myprod.io:9876:trx\n"
+           "    producer3,p2p.blk.example.io:9876:blk\n")
+         ("p2p-bp-gossip-endpoint", boost::program_options::value<vector<string>>()->composing()->multitoken(),
+           "The BP account, inbound connection endpoint, outbound connection IP address. "
+           "The BP account is the producer peer name to retrieve peer-key from on-chain peerkeys table registered on-chain via regpeerkey action. "
```

Comment:
`producer peer name` is confusing. It sounds like a peer name of the producer. Maybe `producer name for peering`.

There are other places in the code using the naming convention, like `my_bp_peer_accounts` in `config_t`. Maybe too late to change.

Reply:
https://github.com/AntelopeIO/spring/pull/1503/commits/2236eb6badb3c46c08188e812cc6528bba341722

Reply:
I am glad it exposed an error. And the new names are much clearer and the code is easier to understand.